apps/designer/app/designer/features/style-panel/shared/use-style-data.ts

```diff
@@ -134,6 +137,18 @@ export const useStyleData = ({
     };
   };
 
+  const deleteProperty = (property: StyleProperty) => {
```

Comment:
removeProperty would be nicer since its called this way in the dom on style object

Reply:
actually, removeProperty is on style object, but delete is on styleMap, which is a newer API, so lets keep it